action.yml

```diff
@@ -55,6 +55,10 @@ inputs:
   storacha-proof:
     description: 'Storacha Base64 encoded proof UCAN with capabilities for the space `w3 delegation create did:key:DID_OF_KEY -c space/blob/add -c space/index/add -c filecoin/offer -c upload/add --base64`'
     required: false
+  storacha-delete-old-builds:
+    description: 'Delete old builds from Storacha before uploading new one. Use "true" or "false" (as strings). Be careful with this option as it will delete all uploads to the space, not just the ones from this build.'
```

Comment:
Is there some key/tag with storacha we could use to ensure only certain items are removed?

Reply:
We should replace it with something that "keeps N latest items" or "purge items older than T" 
See https://github.com/ipshipyard/ipfs-deploy-action/pull/36#discussion_r2296433269